modules/administration-guide/pages/installing-image-puller-on-openshift-using-the-web-console.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {image-puller-name-short} on OpenShift using the web console
+:keywords: administration-guide, image-puller, openshift, operator, operatorhub
+:page-aliases: .:installing-image-puller-on-openshift-using-operatorhub, installing-image-puller-on-openshift-using-operatorhub
+
+[id="installing-image-puller-on-openshift-using-the-web-console_{context}"]
+= Installing {image-puller-name-short} on OpenShift using the web console
+
+This procedure describes how to install the {image-puller-operator-name} on OpenShift using the OpenShift web console.
+
+.Prerequisites
+
+* xref:defining-the-list-of-images-to-pull.adoc[]
+
+* xref:defining-the-memory-parameters-for-the-image-puller.adoc[].
+
+* A web console session with administrative access on the destination OpenShift instance. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/web-console.html[Accessing the web console].
+
+.Procedure
+
+. Install the *{image-puller-operator-name}*. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/operators/admin/olm-adding-operators-to-cluster.html#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console].
+
+. Create a *{image-puller-deployment-name}* `{image-puller-cr-name}` operand from the {image-puller-operator-name}, specifying your values for parameters such as *Images* and *Caching Memory Request*. See https://docs.openshift.com/container-platform/4.10/operators/user/olm-creating-apps-from-installed-operators.html[Creating applications from installed Operators].
```

Comment:
```suggestion
. Create a *{image-puller-deployment-name}* `{image-puller-cr-name}` operand from the {image-puller-operator-name}. See https://docs.openshift.com/container-platform/4.10/operators/user/olm-creating-apps-from-installed-operators.html[Creating applications from installed Operators].
```
Remove `, specifying your values for parameters such as *Images* and *Caching Memory Request*` if those steps are clearly covered in the topic you reference. 

If they are not covered in that topic, consider emphasizing these steps by making them a separate sentence, substeps, or note. In their present form, these steps are easy to dismiss as unimportant.